flutter/lib/src/default_integrations.dart

```diff
@@ -76,6 +81,14 @@ class FlutterErrorIntegration extends Integration<SentryFlutterOptions> {
 
     options.sdk.addIntegration('flutterErrorIntegration');
   }
+
+  @override
+  void close() {
+    /// Restore default
+    FlutterError.onError = defaultOnError;
+    defaultOnError = null;
```

Comment:
what's about:
```
if (defaultOnError != null) {
  FlutterError.onError = defaultOnError;
  defaultOnError = null;
}
```

Reply:
yep, similar to https://github.com/getsentry/sentry-java/blob/4a9b3cc961e74421b8c56b77cc4808c5ddae2d5b/sentry/src/main/java/io/sentry/UncaughtExceptionHandlerIntegration.java#L129-L139

Reply:
Yes but it's a best effort (and undocumented) thing since in some cases it's impossible to "undo".

Reply:
https://github.com/getsentry/sentry-dart/blob/main/dart/lib/src/hub.dart#L218-L221